source/views/student-orgs/list.js

```diff
@@ -83,9 +83,19 @@ export class StudentOrgsView extends React.Component {
           $groupableName: head(startCase(sortableName)),
         }
       })
+      // All orgs
       let sorted = sortBy(withSortableNames, '$sortableName')
       let grouped = groupBy(sorted, '$groupableName')
-      this.setState({orgs: grouped})
+      // New orgs
+      let newOrgs = sorted.filter(org => org.newOrg).map(item => {
+        return {
+          ...item,
+          $groupableName: 'New',
+        }
```

Comment:
Could you just put this whole thing on one line please? return through the end bracket.